src/main/java/org/jitsi/jigasi/TranscriptionGatewaySession.java

```diff
@@ -26,6 +27,7 @@
 import org.jitsi.service.neomedia.*;
 import org.jitsi.service.neomedia.device.*;
 import org.jitsi.util.*;
+import org.jivesoftware.smack.packet.*;
```

Comment:
Should be updated to `import org.jivesoftware.smack.packet.Presence;` as 
`org.jivesoftware.smack.packet.Message` &  `net.java.sip.communicator.service.protocol.Message` results in ambiguity.